### PR TITLE
docs: Rename Twitter to X (formerly Twitter)

### PR DESCRIPTION
**Pull request by Mayureshd-18**
## PR Checklist
Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/angular/angular/blob/main/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [x] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [x] Documentation content changes
- [ ] angular.io application / infrastructure changes
- [ ] Other... Please describe:

## What is the current behavior?

Issue Number: N/A

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -129,7 +129,7 @@ Help us keep Angular open and inclusive. Please read and follow our [Code of Con
 
 Join the conversation and help the community.
 
-- [Twitter][twitter]
+- [X][X]
```
Comment (JeanMeche) [inline README.md:132]: Could you maybe replace `X` with `X (formerly Twitter)` ?

File: README.md [hunk 2]
```diff
@@ -158,7 +158,7 @@ Join the conversation and help the community.
 [node.js]: https://nodejs.org/
 [npm]: https://www.npmjs.com/get-npm
 [codeofconduct]: CODE_OF_CONDUCT.md
-[twitter]: https://www.twitter.com/angular
+[X]: https://www.twitter.com/angular
```
Comment (JeanMeche) [inline README.md:161]: same here !

**Comment by Mayureshd-18** [comment]
@JeanMeche Made the changes. Please review.

**Comment by JeanMeche** [comment]
Thanks for the swift update ! 

Could you please squash your commits and reword the remaining one to follow our [commit message guidelines](https://github.com/angular/angular/blob/main/CONTRIBUTING.md#commit).

**Comment by Mayureshd-18** [comment]
@JeanMeche . Made the requested changes. Thanks.

File: README.md [hunk 3]
```diff
@@ -158,7 +158,7 @@ Join the conversation and help the community.
 [node.js]: https://nodejs.org/
 [npm]: https://www.npmjs.com/get-npm
 [codeofconduct]: CODE_OF_CONDUCT.md
-[twitter]: https://www.twitter.com/angular
+[X (formerly Twitter)]: https://www.twitter.com/angular
```
Comment (alfaproject) [inline README.md:161]: If the URL didn't change then I'd say you can keep this as it was, and change the label in line 132 to:
```
- [X (formely Twitter)][twitter]
```

Disclaimer: I don't work for the Angular team but it just feels right, considering the anchor URL didn't change. Maybe better to wait for feedback from a member of the team

**Comment by Mayureshd-18** [comment]
@JeanMeche Would you please take a look?
Regards

**Review by josephperrott — APPROVED** [review]
LGTM

**Comment by josephperrott** [comment]
@Mayureshd-18 The commit message will need to be updated to match out commit guidelines found [here](https://github.com/angular/angular/blob/main/CONTRIBUTING.md#commit)

**Comment by alxhub** [comment]
This PR was merged into the repository by commit 53d9fd54497aa613a7a6951eaaf46ddd02223ca1.

**Pull request closed without merging**